src/igvfd/schemas/phenotypic_feature.json

```diff
@@ -0,0 +1,88 @@
+{
+    "title": "Phenotypic Feature",
+    "description": "Schema for submitting a phenotypic feature.",
+    "$id": "/profiles/phenotypic_feature.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "feature"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
```

Comment:
if there is no attribution, labs would never be able to submit these? is that intended?

Reply:
Not intended, I will add it in.